pennylane/operation.py

```diff
@@ -119,20 +119,20 @@
 # =============================================================================
 
 
-class Wires(IntEnum):
+class ActOn(IntEnum):
```

Comment:
In the GitHub PR comment this is named `ActsOn`, which I think sounds better 😛

Reply:
Oh yes, I went to ``ActOn`` because I felt there is less grammar. 

Will go back to ``ActsOn``!